code/modules/jobs/job_types/silicon.dm

```diff
@@ -11,9 +11,10 @@ AI
 	selection_color = "#ccffcc"
 	supervisors = "your laws"
 	req_admin_notify = 1
-	minimal_player_age = 30
+	minimal_player_age = 14

```

Comment:
I'm mildly against this being lowered.

Reply:
All other head restrictions have age limit set to 14. Even the HoS which has, from what I can tell, the most restrictive requirements has it set to 14. 30 days for the AI seems anomalous and I can only surmise it as a workaround for not requiring silicon exp.

Reply:
no, it's because it's probably the single hardest role you can play on the station

Reply:
Keep the the age, the ai is the most grief capable role and the hardest to take down ICly when admins aren't on, and the point of its long time is to give value to having an account with good standing.

Our servers base it off of byond account age and not server account age anyways, so its not that big of a deal to lock out byond accounts under a month old for a grief capable role like AI.